insights/parsers/ls_usr_sbin.py

```diff
@@ -0,0 +1,46 @@
+"""
+LsUsrSbin - command ``ls -ln /usr/sbin``
+========================================
+
+The ``ls -ln /usr/sbin`` command provides information for the listing of the
+``/usr/sbin`` directory.
+
+Sample input is shown in the Examples. See ``FileListing`` class for
+additional information.
+
+Sample directory list::
+
+    total 41472
+    -rwxr-xr-x. 1 0  0   11720 Mar 18  2014 accessdb
+    -rwxr-xr-x. 1 0  0    3126 Oct  4  2013 addgnupghome
+    -rwxr-xr-x. 1 0  0   20112 Jun  1  2017 addpart
+    -rwxr-xr-x. 1 0  0  371912 Jan 27  2014 postconf
+    -rwxr-sr-x. 1 0 90  218552 Jan 27  2014 postdrop
+
+Examples:
+
+    >>> "accessdb" in ls_usr_sbin
+    False
+    >>> "/usr/sbin" in ls_usr_sbin
+    True
+    >>> ls_usr_sbin.dir_entry('/usr/sbin', 'accessdb')['type']
+    '-'
+
```

Comment:
@TripleJQK based on the filter you have added below, these examples are invalid.  I think the best approach would be for you to include documentation that clearly states this parser is being filtered and describe how someone using this parser should use `add_filter` to include additional data.  You could also add examples showing how adding a filter would change the parser output.

Reply:
@bfahr - Would you please to check if the new modification could satisify your requests?

Reply:
I believe the example is still confusing because you would never use the `add_filter` command in that manner.